doc/user_manual/optimizer.tex

```diff
@@ -81,7 +82,8 @@ \subsection{Gradient Based Optimizers}
 \end{itemize}
 Note that none of these additional parameters will be provided to the output DataObject by default; they must
 be specifically requested by listing them in the output space when defining the optimizing step's output data
-object.
+object.  Also note that if any of these parameters are not available (for instance, on the first iteration),
+their output value will be set to -1, as this value is nonsensical for the step size and convergence values.
```

Comment:
Yes; originally, I put NaN in these places, but our testing harness for CSV is not compatible with intelligent NaN checking; in fact, it fails a test if NaN is detected in the gold file.

In a day where we have more direct control over the testing harness, we can fix this easily.